sviluppo/review.rst

```diff
@@ -57,57 +62,88 @@ Revisione tecnica
 
-Procedere con eventuali altri commenti.
-Inseriti tutti i commenti fare clic su **Finish your review N** dove N è il numero di commenti inseriti.
+.. image:: ./immagini/github_pr_review_r3.png
 
-.. image:: ./immagini/github_pr_review_r1.png
+Proseguire con eventuali revisioni in altre parti di codice.
+Inseriti tutti i commenti fare clic su **Finish your review N**, dove N è il numero di revisioni effettuate.
+Procedere quindi con l'`Invio della revisione`_.
 
-La maschera che si apre consente di inserire un messaggio e il tipo di azione da eseguire tra **Comment** (per lasciare un semplice commento), **Approve** (per approvare le modifiche proposte dalla PR), **Request changes** (per richiedere delle variazioni alle modifiche proposte).
-Per confermare la revisione fare clic su **Submit review**.
+.. image:: ./immagini/github_pr_review_r4.png
+   :align: center
 
 
 Revisione funzionale
 --------------------
 
-Per fare una revisione funzionale è sufficiente aprire la procedura automatica **ci/runbot**
+Per fare una revisione funzionale aprire la procedura automatica **runboat/build** facendo clic sul relativo *Details*.
+
+.. image:: ./immagini/github_pr_show_runboat.png
+   :align: center
+
+Dopo qualche istante comparirà la maschera di gestione dell’istanza Odoo creata automaticamente per la PR. Per avviarla fare clic sul pulsante *start*.
+
+.. image:: ./immagini/runboat_build_cyan.png
+   :align: center
+
+Attendere che lo sfondo diventi verde, a seconda del carico del servizio il tempo di attesa può variare da qualche secondo a qualche minuto. Se necessario è possibile eventualmente ricaricare la pagina.
 
-.. image:: ./immagini/github_pr_show_runbot.png
+.. image:: ./immagini/runboat_build_grey_green.png
+   :align: center
+   
+Fare clic su *live* per aprire l’istanza Odoo contenente le modifiche introdotte dalla PR.
+
+Le credenziali per accedere sono:
+
+* Email: *admin*
+* Password: *admin*
+
+.. image:: ./immagini/odoo_login.png
+   :align: center
+   
+Eseguire quindi il test funzionale. Una volta completato tornare nella PR e nell’ordine:
+ 
+* fare clic sulla scheda *Files changed*;
+* premere il pulsante **Review changes**;
+* comunicare l'esito dei test procedendo con l'`Invio della revisione`_.
+
+.. image:: ./immagini/github_pr_review_changes.png
+   :align: center
 
-Se è presente il pulsante blu
+Invio della revisione
+---------------------
 
-.. image:: ./immagini/runbot_build_online.png
+Per segnalare solo qualche osservazione, compilare la sezione *Leave a comment* e selezionare l’opzione **Comment**.
 
-vuol dire che un'istanza Odoo contenente le modifiche è pronta per eseguire i test, le credenziali da amministratore per accedere sono:
+.. attention:: Quest’opzione non indica una vera e propria revisione. Il proprio nome utente non comparirà nella pagina principale della PR nella sezione **Reviewers** e il commento non conterà per raggiungere il numero di revisioni necessarie per il merge.
```

Comment:
Viene visualizzato così:
![image](https://github.com/odoo-italia/documentazione/assets/3512779/c893963c-f414-4ad7-ab99-dc93da460b6f)
è voluto?

Reply:
> Viene visualizzato così: ![image](https://private-user-images.githubusercontent.com/3512779/343671367-c893963c-f414-4ad7-ab99-dc93da460b6f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk0NzU1MjMsIm5iZiI6MTcxOTQ3NTIyMywicGF0aCI6Ii8zNTEyNzc5LzM0MzY3MTM2Ny1jODkzOTYzYy1mNDE0LTRhZDctYWI5OS1kYzkzZGE0NjBiNmYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYyNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MjdUMDgwMDIzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjY4MjlkNTRhYjcxODZlZmM2MThiM2I4OWI5MjkxNTgzZWU4MjRlMDc2YmQ3NzcyMWJkOWIyOGRlZjYwMmViMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.FobACVgQ0CoaZ7jfCo49wQaBAawbS_IcGNyJkMicTFc) è voluto?

Le direttive che contengono alcuni specifici tag (attention, note, warning ecc..) vengono rappresentate in modo diverso a seconda dell'ambiente nel quale viene effettuata la compilazione. (es. impostazioni lingua, versioni librerie ecc...)

Ad es. da me in locale compare così, seguendo i requisiti indicati [qui](https://github.com/odoo-italia/documentazione/tree/14.0) nel repo documentazione.

![immagine](https://github.com/odoo-italia/documentazione/assets/227940/464bb39e-927d-4c38-a8e6-26da9d8af62c)

Reply:
Ok chiaro, grazie della spiegazione!